tests/e2e/test.sh

```diff
@@ -89,18 +93,32 @@ if [ "$replicas" -eq 1 ]; then
   exit 1
 fi
 
-echo "Waiting for deployment to scale down back to 0 within 2 minutes"
+# Verify that leader election works by forcing a 120 second apiserver outage
+KIND_NODE=$(kind get nodes --name=substratus-test)
+docker exec ${KIND_NODE} iptables -I INPUT -p tcp --dport 6443 -j DROP
+sleep 120
+docker exec ${KIND_NODE} iptables -D INPUT -p tcp --dport 6443 -j DROP
+
+until kubectl get pods; do
+  echo "Waiting for apiserver to be back up"
+  sleep 1
+done
+
+# rerun kubectl logs because previous one got killed when apiserver was down
+kubectl logs --tail=500 -f deployment/lingo &
```

Comment:
I think you might want `--tail=-1` if you are trying to get at all logs since restart.

Reply:
I didn't want to get all the logs which may already have 1000+ entries and only the last 500. I tried and last 100 wasn't enough but last 500 was more than enough.